pytest_tests/testsuites/network/test_homomorphic_hash.py

```diff
@@ -0,0 +1,180 @@
+import logging
+import random
+import string
+
+import allure
+import pytest
+from cluster_test_base import ClusterTestBase
+from common import NEOFS_ADM_CONFIG_PATH, NEOFS_ADM_EXEC
+from file_helper import generate_file
+from neofs_testlib.cli import NeofsAdm
+from python_keywords.container import (
+    create_container,
+    delete_container,
+    get_container,
+    list_containers,
+    wait_for_container_deletion,
+)
+from python_keywords.neofs_verbs import get_netmap_netinfo, head_object, put_object_to_random_node
+
+logger = logging.getLogger("NeoLogger")
+CONTAINERS_NAME_PREFIX = "homo_hash_container_"
+
+
+@allure.title("Homomorphic hash disabling/enabling")
+@pytest.mark.homo_hash
+class TestHomomorphicHash(ClusterTestBase):
+    @allure.title("Verify homomorphic hash disabling/enabling")
```

Comment:
Check for the default value first?

Reply:
Like check system defaults? It should be another test to run immediately after system deployment (e.g. after dev-env was started). If we add such test here, we assume the system under test is clear, but we can't guarantee it, since any number of tests can be run before these ones, and they can potentially change the system state.